bleep-core/src/scala/bleep/commands/Test.scala

```diff
@@ -18,7 +22,22 @@ case class Test(watch: Boolean, projects: Array[model.CrossProjectName]) extends
   override def runWithServer(started: Started, bloop: BuildServer): Either[BleepException, Unit] =
     DoSourceGen(started, bloop, watchableProjects(started)).flatMap { _ =>
       val targets = BleepCommandRemote.buildTargets(started.buildPaths, projects)
-      val result = bloop.buildTargetTest(new bsp4j.TestParams(targets)).get()
+      val tests = bloop.buildTargetScalaTestClasses(new bsp4j.ScalaTestClassesParams(targets)).get()
+      val testClasses: List[String] = tests.getItems().asScala.flatMap(_.getClasses.asScala).toList
+
+      val testParams = new bsp4j.TestParams(targets)
+
+      testSuites.foreach { classes =>
+        val testClassesData = new bsp4j.ScalaTestSuites(
+          testClasses.filter(classes.toList.contains).map(cls => new bsp4j.ScalaTestSuiteSelection(cls, List.empty[String].asJava)).asJava,
+          List.empty[String].asJava,
```

Comment:
Consider allowing passing jvm options

Reply:
I'd like a follow-up task where we standardize handling of environment variables, java properties and so on between `test`,`run`, `script` (maybe others?) - and then document it properly

Reply:
see #406